x-pack/plugins/alerting/common/execution_log_types.ts

```diff
@@ -18,8 +18,20 @@ export const executionLogSortableColumns = [
   'num_new_alerts',
 ] as const;
 
+export const actionErrorLogSortableColumns = [
+  '@timestamp',
+  'event.start',
+  'event.end',
+  'event.provider',
```

Comment:
Probably don't need to sort by this since event.provider should only be actions.